### PR TITLE
Minor doc improvements (AnimationController)

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

**Comment by Hixie** [comment]
cc @yjbanov for review

File: packages/flutter/lib/src/scheduler/ticker.dart [hunk 1]
```diff
@@ -391,6 +395,12 @@ class TickerFuture implements Future<Null> {
 
   /// A future that resolves when this future resolves or throws when the ticker
   /// is canceled.
+  ///
+  /// If this property is never accessed, then canceling the ticker does not
+  /// throw any exceptions. Once this property is accessed, though, if the
+  /// corresponding ticker is canceled, then an exception will be thrown on the
+  /// [Future] returned by this getter, and if that exception is not caught,
+  /// there will be an uncaught exception in the current zone.
```
Comment (yjbanov) [inline packages/flutter/lib/src/scheduler/ticker.dart]: nit: I think it's more common to say "complete with error" than "throw exception on future" ([1](https://api.dartlang.org/stable/1.24.3/dart-async/Future/Future.error.html), [2](https://api.dartlang.org/stable/1.24.3/dart-async/Completer/completeError.html)), although that might have been before `async`/`await` added the ability to `try`/`catch` future errors.

**Review by yjbanov — APPROVED** [review]

**Comment by yjbanov** [comment]
lgtm

File: packages/flutter/lib/src/animation/animation_controller.dart [hunk 2]
```diff
@@ -465,6 +486,13 @@ class AnimationController extends Animation<double>
   /// [TickerFuture.orCancel] derivative future completes with a [TickerCanceled]
   /// error. By passing the `completed` argument with the value false, this is
   /// reversed, and the futures complete successfully.
+  ///
```
[the remark this answers is not in the document]
Comment (Hixie) [reply packages/flutter/lib/src/animation/animation_controller.dart]: fixed

File: packages/flutter/lib/src/scheduler/ticker.dart (hunk 1 above)
Comment (Hixie) [reply packages/flutter/lib/src/scheduler/ticker.dart]: fixed

**Comment by Hixie** [comment]
Fixed as suggested, thanks for the review. Will land on green.

**Pull request merged**